docs/howto/bill.md

```diff
@@ -77,20 +86,23 @@ Currently this is the recommended way of retrieving the costs from GCP.
 ```
 
 1. Go to the [2i2c billing account] on GCP
-2. Select 'Reports' on the left sidebar
-3. Under time range on the right sidebar, select 'Invoice Month'
-4. Select the time range you are interested in. Note that this has to be at least two months right now, or the next step does not work
-5. Under 'Group by', select 'Month -> Project'.
-6. Under the chart, click the 'Download CSV' button. This downloads a CSV that you can use to later populate the columns in the costs spreadsheet
+1. Select 'Reports' on the left sidebar
+1. Under time range on the right sidebar, select 'Invoice Month'
+1. Select the time range you are interested in. Note that this has to be at least two months right now, or the next step does not work
+1. Under 'Group by', select 'Month -> Project'.
+1. Under the chart, click the 'Download CSV' button. This downloads a CSV that you can use to later populate the columns in the costs spreadsheet
    ```{figure} ../images/gcp-billing-ui.png
    GCP billing UI
    ```
-7. Copy GCP costs.
-
-   The GCP CSV has rows for each month and project. Carefully copy the $ values under
-   'subtotal' (last column) into the [spreadsheet for billing]. Match the 'project name' column in the GCP CSV with the 'cloud project' row in our billing spreadsheet.
+1. Run the following deployer command to convert the generated CSV file into the format required for the invoicing process.
+   
+   ```bash
+   deployer transform cost-table gcp pathto/downloaded/csvfile
+   ```
 
-   When done, re-check the numbers once again to make sure they are copied correctly.
+   This will output a new CSV file to your local filesystem called `2i2c_dedicated_cluster_billing_GCP_{START_MONTH}_{END_MONTH}.csv`.
+1. Upload this CSV file to the [cloud costs folder]
```

Comment:
Same comment as earlier about filename.

Reply:
Resolved in [`4457d0b` (#4063)](https://github.com/2i2c-org/infrastructure/pull/4063/commits/4457d0b77f8ae04290021d61ffd16394ea6d087f)